options_test.go

```diff
@@ -63,14 +63,23 @@ var _ = Describe("Options", func() {
 		})
 	})
 
+	Describe("#ChromeOptions", func() {
+		It("should return an Option with ChromeOptions set", func() {
+			config := NewTestConfig()
+			ChromeOptions("args", []string{"v1", "v2"})(config)
+			Expect(config.ChromeOptions["args"]).To(Equal([]string{"v1", "v2"}))
+		})
+	})
+
 	Describe("#Merge", func() {
 		It("should apply any provided options to an existing config", func() {
 			config := NewTestConfig()
 			Browser("some browser")(config)
-			newConfig := config.Merge([]Option{Timeout(5), Debug})
+			newConfig := config.Merge([]Option{Timeout(5), Debug, ChromeOptions("args", "value")})
```

Comment:
`Merge` is very simple and doesn't have code that specifically deals with `ChromeOptions`, so this test doesn't really need to change.